### ISSUE TITLE
No way to recover unsaved file contents

### ISSUE THREAD

**Issue by Eikix** [AstroNvim/AstroNvim#1448]
### Checklist

- [X] Using a stable version of Neovim (i.e. not neovim nightly)
- [X] `:PackerSync`
- [X] `:AstroUpdate`
- [X] Restarted AstroNvim

### AstroNvim version

v2.9.1

### Neovim version (>= 0.8)

v0.8.1

### Operating system/version

macOS Ventura 13.0

### Describe the bug

Hi! I recently installed NeoVim along with AstroVim. 

I was editing a file this morning when I accidentally pressed CMD + W (macOS), thus closing my terminal instance (iTerm2).
Upon reopening my file. I noticed I had lost all my changes (forgot to press CTRL + S to save).

The `:rec` command did not work.
There isn't a `main.rs~` edit file nor a `.swp` file.

### Steps to Reproduce

1. open iTerm2 on macOS ventura 13.0
2. open a `main.rs` in a Rust project (initialised using `cargo new <name_of_the_project>` and `cd` into it.
3. start editing it
4. press CMD + W before saving the file.

### Expected behavior

I expected to be able to recover my unsaved changes.

### Additional Context

I have not created a new user config. I'm using the default AstroVim configuration.

**Comment by mehalter** [comment]
By default we are disabling `swapfile`, I have also been thinking about changing this default because it can be pretty detrimental. I have gone ahead and changed this on `nightly` and will get it included in the v2.10.0 release later this week. In the mean time if you don't want to use `nightly` and stick with stable, you can put the following in your `user/init.lua` file:

```lua
return {
  options = {
    opt = {
      swapfile = true,
    }
  }
}
```

**Comment by Eikix** [comment]
Thanks for your quick response and help:D! 
So if I understood correctly: I should create a custom user as per https://astronvim.github.io/Configuration/manage_user_config and then copy paste

```lua
return {
  options = {
    opt = {
      swapfile = true,
    }
  }
}
```
